src/volue/mesh/examples/run_inflow_calculation_async.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+from datetime import datetime
+
+from volue import mesh
+import volue.mesh.aio
+
+from volue.mesh.examples import _get_connection_info
+
+
+async def main(address, port, root_pem_certificate):

```

Comment:
Just wanted to mention we are not too consistent in the sync vs async examples. Some times we create separate files and some time swe do it within one file, like in `authorization.py` or `get_version.py`. Personally I like it to be in one file. (possibly more re-use of parameters, and we have the functionality example in one place). Not a blocker, just mentioning.

Reply:
Ah, nice! I just looked at the files. I'll do the one file thing then, makes things a bit easier.